### ISSUE TITLE
some code need "script.skin.helper.service" ?

### ISSUE THREAD

**Issue by Nanomani** [beatmasterRS/skin.arctic.zephyr.mod#448]
if you search in all files .xml "SkinHelper." you can found some code with this.

![1](https://user-images.githubusercontent.com/3939543/128196640-eede1198-cbae-4bbb-aa34-78a331f826d5.jpg)

this mean that addon "script.skin.helper.service" need to be installed ? right ?

Repo : https://github.com/kodi-community-addons/repository.marcelveldt
Wiki : https://github.com/kodi-community-addons/script.skin.helper.service/wiki

there are functions that would be useful :)
(https://github.com/kodi-community-addons/script.skin.helper.service/wiki/General-window-properties)

eg : 
display count of Favourites in the header
`SkinHelper.TotalFavourites`

Not sure you want this ? :D  
I prefer to warn you

**Comment by beatmasterRS** [comment]
Nope, there will be no more official support for shs, scince it slows down the skin...

**Comment by Nanomani** [comment]
ok ok :D  
just point out some cleaning is needed. Right ? :)

**Comment by semool** [comment]
No dont clean the code. There are users that manual install SHS to use some of the features like animated Posters.

**Comment by Nanomani** [comment]
thanks for explain.

**Comment by beatmasterRS** [comment]
> No dont clean the code. There are users that manual install SHS to use some of the features like animated Posters.

Nothing is to be removed either. It's just about not adding any more supporting code. So all good
